### DISCUSSION TITLE
is it possible to get starter-battery charge status/level?

### DISCUSSION THREAD

**Discussion by pejotigrek** [DurgNomis-drol/ha_toyota#155] [Q&A]
let me explain: from time to time my in-laws park their 2020 Yaris Hybrid in my garage, mostly when they go for a longer vacation. couple of times, after about 3 or 4 weeks while being unused - the starter-battery just died [not the EV one used for driving, just the small one used for ignitiion and "regular" gasoline-based drive]. of course we jump-started the car and everything went smoothly, but I am thinking if I could add in-laws' car to my Home Assistant and monitor the starter-battery for the future knowing if it is on the edge and needs some road trip to charge ;)

I don't want to fiddle with MyT API on their account if it won't help, so I'm asking here before any integration configuration started on my side.

hope I made my question clear :)

**Comment by DurgNomis-drol** [comment]
You would need to add your in-laws car to your Toyota account for it to work. There is no other way to do that. Or you would need to set this up twice with their account on your HA installation.

**Reply by pejotigrek** [reply]
hi! thanks for the answer, but that wasn't exactly what I was asking for - I probably wasn't as clear as I thought ;)
sure I need to add the car to my account or use their account in HA, I tried that, but I was wondering if the starter-battery charge sensor could be a thing - right now, when I see the informations pulled from MyT to HA, there's no battery info.
I'm guessing that this car model doesn't send such informations?

**Reply by DurgNomis-drol** [reply]
It dependes on the car what data is available, my Toyota Aygo have a starter battery sensor.
